### PR TITLE
add new strings per PR #4597

### PR DESCRIPTION
**Pull request by peiying2**

### CODE REVIEW

**Comment by flodolo** [comment]
You missed at least 3 new strings
https://github.com/mozilla/bedrock/pull/4597/files#diff-dfcc4dd366ddd6697c2ddf03b6996429R143

My suggestion would be to try extracting the strings from the template (bedrock/firefox/templates/firefox/new/scene1.html) and see how many are not available in new/horizon.lang or shared files.

As commented on IRC, we should expose strings also for 
https://github.com/mozilla/bedrock/pull/4618

**Comment by peiying2** [comment]
Add the missing strings to the main.lang file.

**Comment by flodolo** [comment]
> Add the missing strings to the main.lang file.

These strings need to be in the same file (horizon.lang). The diff is not helping noticing it
https://github.com/alexgibson/bedrock/blob/d7a43ba1e8843ffa51ea1d2bf80e8b57664e974d/bedrock/firefox/templates/firefox/new/scene1.html#L121-L151

File: en-US/firefox/dnt.lang [hunk 1]
```diff
@@ -82,6 +83,12 @@ How do I enable Do Not Track in Firefox?
 This feature is not enabled by default. You can find the Do Not Track request on the Privacy pane. <span id="%(id)s">On Windows, go to Tools &gt; Options&hellip; &gt; Privacy.</span>
 
 
+## TAG: macos_branding
+;On macOS, go to Firefox &gt; Preferences… &gt; Privacy. On Linux, go to Edit &gt; Preferences &gt; Privacy.
+On macOS, go to Firefox &gt; Preferences… &gt; Privacy. On Linux, go to Edit &gt; Preferences &gt; Privacy.
+
+
+## String is obsolte.  Do NOT remove.
```
Comment (flodolo) [inline en-US/firefox/dnt.lang]: Comment use one #, also typo, and one space after periods.

File: en-US/firefox/new/horizon.lang [hunk 2]
```diff
@@ -3,12 +3,19 @@
...
 ;Free Web Browser
 Free Web Browser
 
 
+## TAG: macos_branding
+;Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows, macOS, Linux, Android and iOS today!
+Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows, macOS, Linux, Android and iOS today!
+
+
+## String is obsolte. Do NOT remove.
```
Comment (flodolo) [inline en-US/firefox/new/horizon.lang]: Same here, minus double space

File: en-US/firefox/new/horizon.lang [hunk 3]
```diff
@@ -123,9 +130,9 @@ Firefox for other platforms
 Download Firefox <span>for Windows</span>
 
 
-## TAG: firefox_new_other_platforms
-;Download Firefox <span>for OS X</span>
-Download Firefox <span>for OS X</span>
+## TAG: macos_branding
```
Comment (flodolo) [inline en-US/firefox/new/horizon.lang]: This needs to remain behind the tag firefox_new_other_platforms

File: en-US/thunderbird/features.lang [hunk 4]
```diff
@@ -146,6 +147,12 @@ Thunderbird protects you from email scams which try to trick users into handing
 Automated Update
 
 
+## TAG: macos_branding
+;Thunderbird’s update system checks to see if you’re running the latest version, and notifies you when a security update is available. These security updates are small (usually 200KB - 700KB), giving you only what you need and making the security update quick to download and install. The automated update system provides updates for Thunderbird on Windows, macOS, and Linux in over 40 different languages.
+Thunderbird’s update system checks to see if you’re running the latest version, and notifies you when a security update is available. These security updates are small (usually 200KB - 700KB), giving you only what you need and making the security update quick to download and install. The automated update system provides updates for Thunderbird on Windows, macOS, and Linux in over 40 different languages.
+
+
+## String is obsolete.  Do NOT remove.
```
Comment (flodolo) [inline en-US/thunderbird/features.lang]: Double pound, double space.

File: en-US/firefox/new/horizon.lang (hunk 3 above)
Comment (peiying2) [inline en-US/firefox/new/horizon.lang]: My thinking on this is that since this has not been exposed, so I thought we can remove it all together.

Comment (peiying2) [inline en-US/firefox/new/horizon.lang]: Or are you suggesting that the macos_branding behind all other firefox_new_other_platforms tags?

Comment (flodolo) [inline en-US/firefox/new/horizon.lang]: No. The string `Download Firefox <span>for OS X</span>` should be behind the `firefox_new_other_platforms` tag, then we should have `Download Firefox <span>for OS X</span>` behind `macos_branding`.

Since we never exposed the first string, we can expose it directly behind the original tag `firefox_new_other_platforms`. 

Imagine a situation where locales translate the macOS strings, including this one, but not the others. We still couldn't show that string, could we?

TL;DR this should be
```
## TAG: firefox_new_other_platforms
;Download Firefox <span>for macOS</span>
Download Firefox <span>for macOS</span>
```

The rest of the file is OK.

File: en-US/firefox/new/horizon.lang [hunk 5]
```diff
@@ -104,3 +118,28 @@ Freedom is yours
...
+
+
+## TAG: firefox_new_other_platforms
+;Download Firefox <span>for Windows</span>
+Download Firefox <span>for Windows</span>
+
+
+## TAG: macos_branding
+;Download Firefox <span>for macOS</span>
+Download Firefox <span>for macOS</span>
```
Comment (flodolo) [inline en-US/firefox/new/horizon.lang]: You have the string twice now, you need to remove this one.

Comment (peiying2) [inline en-US/firefox/new/horizon.lang]: Are you saying since the purpose of the tag for macos_branding is to update OS X to macOS, from l10n perspective, the actual string to be localized at this point is the same between the two tagged strings, which is "Download Firefox <span>for macOS</span>".   Tag firefox_new_other_platforms will take care of the update.

Comment (flodolo) [inline en-US/firefox/new/horizon.lang]: By design a lang file can't have the same string twice. The string needs to be there only once, and in the code it's behind the other tag for practical reasons (not the macos_branding one).
